src/commands/root.command.ts

```diff
@@ -14,26 +14,46 @@ const example = chalk.green
 const helptext = `
 Example Usage
 
-  $ ${example('topos-zkevm-demo init')}
-    Initialize the Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
+  $ ${example('topos-zkevm-demo install')}
+    Install Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
```

Comment:
nit: do we want to specify the path of the log file?

Reply:
The path is detailed in the output of the command